[2023-04-05T04:39:54.769+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ELT.object_transformation.sql manual__2023-04-05T04:39:49.469443+00:00 [queued]>
[2023-04-05T04:39:54.781+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ELT.object_transformation.sql manual__2023-04-05T04:39:49.469443+00:00 [queued]>
[2023-04-05T04:39:54.782+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-05T04:39:54.782+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-05T04:39:54.782+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-05T04:39:54.794+0000] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): object_transformation.sql> on 2023-04-05 04:39:49.469443+00:00
[2023-04-05T04:39:54.800+0000] {standard_task_runner.py:55} INFO - Started process 249 to run task
[2023-04-05T04:39:54.803+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'ELT', 'object_transformation.sql', 'manual__2023-04-05T04:39:49.469443+00:00', '--job-id', '83', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmpq1q_97r4']
[2023-04-05T04:39:54.804+0000] {standard_task_runner.py:83} INFO - Job 83: Subtask object_transformation.sql
[2023-04-05T04:39:54.869+0000] {task_command.py:389} INFO - Running <TaskInstance: ELT.object_transformation.sql manual__2023-04-05T04:39:49.469443+00:00 [running]> on host f5e703d34d7f
[2023-04-05T04:39:54.935+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=200300600@ucaribe.edu.mx
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ELT
AIRFLOW_CTX_TASK_ID=object_transformation.sql
AIRFLOW_CTX_EXECUTION_DATE=2023-04-05T04:39:49.469443+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-05T04:39:49.469443+00:00
[2023-04-05T04:39:54.940+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-04-05T04:39:54.941+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'dbt run --project-dir demo']
[2023-04-05T04:39:54.957+0000] {subprocess.py:86} INFO - Output:
[2023-04-05T04:39:57.556+0000] {subprocess.py:93} INFO - [0m04:39:57  Encountered an error:
[2023-04-05T04:39:57.557+0000] {subprocess.py:93} INFO - Runtime Error
[2023-04-05T04:39:57.557+0000] {subprocess.py:93} INFO -   fatal: Invalid --project-dir flag. Not a dbt project. Missing dbt_project.yml file
[2023-04-05T04:39:57.818+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-04-05T04:39:57.833+0000] {taskinstance.py:1776} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-04-05T04:39:57.837+0000] {taskinstance.py:1332} INFO - Marking task as FAILED. dag_id=ELT, task_id=object_transformation.sql, execution_date=20230405T043949, start_date=20230405T043954, end_date=20230405T043957
[2023-04-05T04:39:57.853+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 83 for task object_transformation.sql (Bash command failed. The command returned a non-zero exit code 2.; 249)
[2023-04-05T04:39:57.907+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-04-05T04:39:57.929+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
